README.md

```diff
@@ -82,9 +83,10 @@
 }
 ```
 
-| Method | URL                      | Description                                                         |
-| ------ | ------------------------ | ------------------------------------------------------------------- |
-| [GET]  | /instructor/courses/ | Returns an array containing all course event objects belonging to the currently logged in instructor. |
+| Method | URL                            | Description                                                                                           |
+| ------ | ------------------------------ | ----------------------------------------------------------------------------------------------------- |
+| [GET]  | /instructor/courses/           | Returns an array containing all course event objects belonging to the currently logged in instructor. |
+| [GET]  | /instructor/profile/profile_id | Returns a profile object of a specific instructor .                                                   |
```

Comment:
Changing /instructor/profile/profile_id to /instructor/profile/:profile_id could help more quickly understand that you need to pass the profile_id for the endpoint